main.rb

```diff
@@ -0,0 +1,182 @@
+require './school-library/student'
+require './school-library/rental'
+require './school-library/persons'
+require './school-library/teacher'
+require './school-library/book'
+
+class Methods
+  def initialize
+    @person_array = []
+    @books = []
+    @rental_array = []
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_option = Integer(gets.chomp)
+
+    case person_option
+    when 1
+      print 'Age: '
+      user_age = Integer(gets.chomp)
+
+      print 'Name: '
+      user_name = gets.chomp
+
+      print 'Has parent permission? [Y/N]: '
+      user_permission = gets.chomp.to_s.upcase
+
+      case user_permission
+      when 'Y'
+        user_permission = true
+      when 'N'
+        user_permission = false
+      end
+      student = Student.new(user_age, user_name, parent_permission: user_permission)
+      @person_array.push({
+                           output: "[Student] Name: #{student.name}, ID: #{student.id}, Age: #{student.age}",
+                           object: student
+                         })
+
+      puts 'Person created successfully!'
+      puts "\n"
+    when 2
+      print 'Age: '
+      user_age = Integer(gets.chomp)
+
+      print 'Name: '
+      user_name = gets.chomp
+
+      print 'Specialization: '
+      user_specialization = gets.chomp
+
+      teacher = Teachers.new(user_age, user_name, user_specialization)
```

Comment:
When I was trying to create a teacher your application threw an error after typing the `Specialization`. Please correct this bug, having your application free of bugs will help your projects look more professional because you have attention to detail.

Look at the image below for a better understanding:
![Capture](https://user-images.githubusercontent.com/75709904/149530563-82cc0207-3f56-46ce-bbb3-fae4bd7eb77b.PNG)